ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.20
+    tag: golang-1.21
```

Comment:
@jiridanek FYI in case these are useful images for you to use versus what you've got in https://github.com/openshift/release/pull/53232/commits/e2bd3ac2dc9a191d886c09c8b65dc813020dc6c5#diff-5cc664e1406d2eb67075795f9e2edb31287894734282634c780e1fa45f68c854R26

Reply:
Thanks! Yeah, this looks like the right thing to do, as opposed to what I have. Thanks again.